test/tile/raster_tile.test.cpp

```diff
@@ -15,6 +16,7 @@ class RasterTileTest {
 public:
     FakeFileSource fileSource;
     TransformState transformState;
+    util::RunLoop loop;
```

Comment:
Tests were missing an explicit `RunLoop`. If running these tests separately, this is not an issue because we had the default `RunLoop`, but when running after other unit tests that create/destroy a `RunLoop`, we got a null `RunLoop` further down the line.